spec/requests/providers/applicants_spec.rb

```diff
@@ -21,9 +21,20 @@
         expect(response).to redirect_to(providers_legal_aid_applications_path)
       end
     end
+
+    context 'has already got applicant infro' do
```

Comment:
**[nit pick/typo]** _info_ (instead of _infro_). 

The phrasing itself looks a bit odd IMO, would something like _when the application already has applicant info_ be a bit more clearer?